lib/apple/index.js

```diff
@@ -89,6 +89,10 @@ exports.verifyPayment = function (payment, cb) {
 		});
 	}
 
+	if (typeof payment.secret !== 'undefined') {
```

Comment:
True for variable names yeah. But on properties you don't need that. I prefer the change if you don't mind. Thanks :)